linkerd-tcp/prometheus.yml

```diff
@@ -0,0 +1,25 @@
+global:
+  scrape_interval:     10s
+  evaluation_interval: 10s
+
+scrape_configs:
```

Comment:
i think you've already thought about this, but in case not, consider whether any interesting grafana graphs can be rendered from slow_cooker data.